### PR TITLE
Worked on asset management modal and changed data is to data are.

### PR DESCRIPTION
**Pull request by RyanMaciel**

### CODE REVIEW

File: ooiui/static/js/views/asset_management/AssetView.js [hunk 1]
```diff
@@ -6,13 +6,20 @@ var ParentAssetView = Backbone.View.extend({
      *  - derender()
      */
     initialize: function() {
-        _.bindAll(this, 'render', 'derender');
+        "use strict";
```
Comment (birdage) [inline ooiui/static/js/views/asset_management/AssetView.js]: this should just go to the top of the file, remove multiple instances

Comment (FBRTMaka) [inline ooiui/static/js/views/asset_management/AssetView.js]: @birdage , actually - not the case.  All "use strict"; should be declared in each function.

Comment (FBRTMaka) [inline ooiui/static/js/views/asset_management/AssetView.js]: @birdage , at least that's what Lint says lol
screen shot 2015-11-04 at 3 10 56 pm

Comment (birdage) [inline ooiui/static/js/views/asset_management/AssetView.js]: @FBRTMaka each to each own...

Comment (FBRTMaka) [inline ooiui/static/js/views/asset_management/AssetView.js]: @birdage but that's F**** S****...so lets move it to the top and only do it once.  LOL

**Pull request closed without merging**